yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -579,7 +579,7 @@ export class Archiver implements ArchiveSource {
    * @param blockNumber - L2 block number to get messages for.
    * @returns The L1 to L2 messages/leaves of the messages subtree (throws if not found).
    */
-  getNewL1ToL2Messages(blockNumber: bigint): Promise<Buffer[]> {
+  getNewL1ToL2Messages(blockNumber: bigint): Promise<Fr[]> {
```

Comment:
Originally had it as buffer because I didn't realize that we are already truncating the content hash to 1 field (I assumed we represent it as 2 fields as well).